build/follow-me/view.js.map

```diff
@@ -0,0 +1 @@
efinition) => {\n\tfor(var key in definition) {\n\t\tif(__webpack_require__.o(definition, key) && !__webpack_require__.o(exports, key)) {\n\t\t\tObject.defineProperty(exports, key, { enumerable: true, get: definition[key] });\n\t\t}\n\t}\n};","__webpack_require__.o = (obj, prop) => (Object.prototype.hasOwnProperty.call(obj, prop))","// define __esModule on exports\n__webpack_require__.r = (exports) => {\n\tif(typeof Symbol !== 'undefined' && Symbol.toStringTag) {\n\t\tObject.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });\n\t}\n\tObject.defineProperty(exports, '__esModule', { value: true });\n};","// no baseURI\n\n// object to store loaded and loading chunks\n// undefined = chunk not loaded, null = chunk preloaded/prefetched\n// [resolve, Promise] = chunk loading, 0 = chunk loaded\nvar installedChunks = {\n\t\"follow-me/view\": 0,\n\t\"follow-me/style-view\": 0\n};\n\n// no install chunk\n\n// no chunk on demand loading\n\n// no prefetching\n\n// no preloaded\n\n// no external install chunk\n\n__webpack_require__.O.j = (chunkId) => (installedChunks[chunkId] === 0);","","// startup\n// Load entry module and return exports\n// This entry module depends on other loaded chunks and execution need to be delayed\nvar __webpack_exports__ = __webpack_require__.O(undefined, [\"follow-me/style-view\"], () => (__webpack_require__(\"./src/follow-me/view.js\")))\n__webpack_exports__ = __webpack_require__.O(__webpack_exports__);\n",""],"names":["isCssVariableDefined","variableName","window","getComputedStyle","styles","document","documentElement","value","getPropertyValue","trim","getBackgroundColor","color","varName","background","getLinkColor","text","match","substring","split","generateSelector","selector","prop","pseudo","getStyles","button","hover","getBlockStyles","base","style","backgroundColor","buttonTextColor","buttonColor","elements","link","buttonHoverColor","getPopupStyles","store","getContext","apiFetch","wp","trapFocus","element","focusableElements","querySelectorAll","firstFocusableElement","lastFocusableElement","length","focus","addEventListener","event","key","keyCode","shiftKey","activeElement","preventDefault","state","actions","openModal","context","isModalOpen","body","classList","add","setTimeout","blockWrapper","getElementById","blockId","modalFrame","querySelector","closeModal","remove","openButton","toggleModal","copyToClipboard","navigator","clipboard","writeText","webfinger","then","copyButtonText","i18n","copied","copy","error","console","updateRemoteProfile","remoteProfile","target","isError","errorMessage","handleKeyDown","submitRemoteProfile","namespace","input","emptyProfileError","test","invalidProfileError","isLoading","path","userId","encodeURIComponent","response","open","url","message","genericError","callbacks","initButtonStyles","buttonStyle","styleElement","createElement","textContent","head","appendChild","popupStyleElement","documentKeydown","documentClick","toggleButton","contains"],"sourceRoot":""}
```

Comment:
where does the map files come from? is that a config change?

Reply:
I think they're just left-overs from early dev mode. Good eye!
Removed in [c6b54f2](https://github.com/Automattic/wordpress-activitypub/pull/1691/commits/c6b54f247e171b4b2087b6bf04f9fdb7bb3fc980).